requirements.txt

```diff
@@ -1,10 +1,13 @@
 certifi==2019.3.9
 chardet==3.0.4
-Django~=2.0.3
+Django==2.0.13
 django-bootstrap3==9.1.0
+django-filter==2.2.0
 django-reversion==2.0.13
+djangorestframework==3.10.3
 gunicorn==19.7.1
 idna==2.8
+Markdown==3.1.1
 pytz==2019.1
 requests==2.21.0
-urllib3==1.24.2
+urllib3==1.24.2
```

Comment:
I would have expected to only see `djangorestframework` added here.
Could you please explain the other changes?

Reply:
- Markdown & django-filter: [recommended packages here](https://www.django-rest-framework.org/#installation) and included in exodus, but I can probably remove both (Not using filters for now, and could remove the browsable api)
- Django version change: fixing the version installed when running with `~=`

Reply:
Thanks for "markdown & django-filter", I'd say to include them only if we think we need them.

Note that I didn't ask about the django version change :)

Reply:
Ok removed